.gitignore

```diff
@@ -18,6 +18,7 @@ node_modules
 .env.development.local
 .env.test.local
 .env.production.local
+.history
```

Comment:
where is this from? What's the reason?

Reply:
That's a folder created by VS Code history plugin. I guess it might be useful to have it here since there are already ".DS_Store" and ".idea" folders listed.